infrastructure/modules/resource_templates/traffic_manager/main.tf

```diff
@@ -0,0 +1,41 @@
+data "azurerm_resource_group" "resource_group" { name = var.resource_group_name }
+
+resource "azurerm_public_ip" "public_ip" {
+  name                = var.public_ip_name
+  location            = data.azurerm_resource_group.resource_group.location
+  resource_group_name = data.azurerm_resource_group.resource_group.name
+  allocation_method   = "Static"
+  domain_name_label   = var.public_ip_name
+  sku                 = "Standard"
+}
+
+resource "azurerm_traffic_manager_profile" "traffic_manager_profile" {
+  name                   = var.name
+  resource_group_name    = data.azurerm_resource_group.resource_group.name
+  traffic_routing_method = "Weighted"
+
+  dns_config {
+    relative_name = var.name
+    ttl           = 100
+  }
+
+  monitor_config {
+    protocol                     = "HTTP"
+    port                         = 80
+    path                         = "/"
+    interval_in_seconds          = 30
+    timeout_in_seconds           = 9
+    tolerated_number_of_failures = 3
+  }
+
+  tags = {
+    environment = "Production"
```

Comment:
Does this mean this is the legit 'Production' environment or something else?  (Tell me if I'm being dumb)

Reply:
@guypritchard - this is handled in the next PR that @adelechurchill and I have worked on. See this: https://github.com/buerokratt/Infrastructure/blob/8b5594ce3fd0d0d9ac25c057b35a74cf719395ce/infrastructure/modules/traffic_manager/main.tf#L25